pom.xml

```diff
@@ -144,6 +144,11 @@
 			<artifactId>junit-vintage-engine</artifactId>
 			<version>${junit-jupiter.version}</version>
 		</dependency>
+		<dependency>
+            <groupId>org.junit.platform</groupId>
+            <artifactId>junit-platform-commons</artifactId>
+            <version>1.5.2</version>
+ 		</dependency>
```

Comment:
How does this new dependency come into play?

Reply:
This dependency solves the following exception when running unit tests in Eclipse
`java.lang.NoClassDefFoundError: org/junit/platform/commons/PreconditionViolationException`

Reply:
Oh I see. So I guess we would not encounter that issue in Idea?